base/Makefile

```diff
@@ -68,7 +68,11 @@ ifeq ($(XC_HOST),)
 else
 	@echo "const MACHINE = \"$(XC_HOST)\"" >> $@
 endif
-	@echo "const libm_name = \"$(LIBMNAME)\"" >> $@
+ifeq ($(USE_SYSTEM_LIBM),0)
+	@echo "const libm_name = \"$(LIBMNAME).$(SHLIB_EXT)\"" >> $@
+else
+	@echo "const libm_name = \"$(LIBMNAME).$(SHLIB_EXT).$(LIBM_VERSION)\"" >> $@
```

Comment:
```jl
julia> versioninfo()
Julia Version 1.9.0-DEV.876
Commit e2e14cbdb9* (2022-07-03 20:35 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 6 × Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz
  WORD_SIZE: 64
  LIBM: libm.so.6
  LLVM: libLLVM-14.0.5 (ORCJIT, skylake)
  Threads: 1 on 6 virtual cores
```
https://github.com/JuliaLang/julia/blob/dd5760deb71e572f6713ad543b2472331952660e/stdlib/InteractiveUtils/src/InteractiveUtils.jl#L142

Do we need to show suffixes and ABI versions?
Perhaps the library name is enough.

Reply:
Maybe keep `libm_name = "libm"`.
And add a new var `libm_version` like `libllvm_version`. 

https://github.com/JuliaLang/julia/blob/dd5760deb71e572f6713ad543b2472331952660e/stdlib/InteractiveUtils/src/InteractiveUtils.jl#L143

Reply:
That's a good idea, but how do we deal with the `.so` suffix ? Also `libllvm_version` is a full `VersionNumber` and we only need the major number ... We could add a `const libm_suffix = ".so.6"`, but I don't know if there will be consensus on this ...

Reply:
As an alternative, we could keep `libm_name = "libm.so.6"` and use `replace(Base.libm_name, r"\.[^\.]+"=>"")` in `InteractiveUtils`.